### PR TITLE
Create COMMAND_SUMMARY_ALPHABETIC.MD doc'n file

### PR DESCRIPTION
**Pull request by PeterCon**
This may be a helpful resource for people learning to use StoreBroker. This is a start; I would add key input/return values for more commands as well as links to the Store submission API docs. A second page that lists commands in logical groupings might also be useful.

### CODE REVIEW

File: Documentation/COMMAND_SUMMARY_ALPHABETIC.MD [hunk 1]
```diff
@@ -16,24 +16,41 @@ For some commands, links are provided to documentation for the corresponding RES
 related resources/objects.
 
 ## Clear-StoreBrokerAuthentication
-> Clears out any cached tenantId, client id, and client secret credential from this PowerShell session. Also disables usage of the proxy server if that had been previously enabled. All future remote commands from this module will once again prompt for credentials.
+
+<dl><dd>
```
Comment (danbelcher-MSFT) [inline Documentation/COMMAND_SUMMARY_ALPHABETIC.MD]: Are these tags intentional?  I don't see them affecting the rendered markdown in any way.

Comment (PeterCon) [reply Documentation/COMMAND_SUMMARY_ALPHABETIC.MD]: They cause the content in between to be indented a bit. Given a long list, the formatting helps in creating a visual information hierarchy.

Comment (danbelcher-MSFT) [reply Documentation/COMMAND_SUMMARY_ALPHABETIC.MD]: Thanks, I see the effect now.

**Comment by danbelcher-MSFT** [comment]
Hey Peter, is this ready for review?  Just let us know and we'll take a look.  Thanks!

**Pull request closed without merging**